src/encord_active/cli/main.py

```diff
@@ -123,11 +124,17 @@ def import_local_project(
         help="The root directory of the dataset you are trying to import",
         file_okay=False,
     ),
-    glob: List[str] = typer.Option(
+    data_glob: List[str] = typer.Option(
         ["**/*.jpg", "**/*.png", "**/*.jpeg", "**/*.tiff"],
-        "--glob",
-        "-g",
-        help="Glob pattern to choose files. Note that you can repeat the `--glob` argument if you want to match multiple things.",
+        "--data-glob",
+        "-dg",
+        help="Glob pattern to choose files. Repeat the `--data-glob` argument to match multiple globs.",
+    ),
+    label_glob: List[str] = typer.Option(
+        None,
+        "--label-glob",
+        "-lg",
+        help="Glob pattern to choose label files. Repeat the `--label-glob` argument to match multiple globs. This argument is only used if you also provide the `transformer` argument",
```

Comment:
I agree. It would be great to rewrite to COCO parser to do this. We could even have two COCO parsers - one for segmentations and one for bounding boxes. That would reduce the ambiguity between the two types of labels that are currently in the importer.